ocrd_tool.md

```diff
@@ -19,6 +19,27 @@ Optionally, workflow processors can be notified that this file is potentially
 large and static (e.g. a fixed dataset or a precomputed model) and should be cached
 indefinitely after download by setting the `cacheable` property to `true`.
 
+The filename itself, i.e. the concrete value `<fpath>` of a file parameter
+should be resolved in the following way:
+
+* If `<fpath>` is an `http`/`https` URL: Download to a temporary directory (if
+  `cacheable==False`) or a semi-temporary cache directory (if `cacheable==True`)
+* If `<fpath>` is an absolute path: Use as-is.
+* If `<fpath>` is a relative path, try resolving the following paths and return
+  the first one found if any, raise a `FileNotFoundError` otherwise:
+  * `$CWD/<fpath>`
+  * If an environment variable is defined that has the name of the processor in
+    upper-case and with `-` replaced with `-` and followed by `_PATH` (e.g. for a processor
+    `ocrd-dummy`, the variable would need to be called `OCRD_DUMMY_PATH`):
+    * Split the variable value at `:` and try to resolve by prepending each token
+      with `<fname>`
```

Comment:
I don't understand this. What is `fname` here?

Reply:
Was from a previous revision, should read:

> Split the variable value at : and try to resolve by appending `<fpath>`
> to each token and return the first found file if any.

Reply:
Oh, now I got it. List-valued like `PATH`